### PR TITLE
some fail

### PR DESCRIPTION
**Pull request by DenisGurskiy**
https://DenisGurskiy.github.io/react_todo-app-with-api/

### CODE REVIEW

File: src/components/TodoItem/TodoItem.tsx [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+//             placeholder="Empty todo will be deleted"
+//             value="Todo is being edited now"
+//           />
+//         </form>
+
+//         <div data-cy="TodoLoader" className="modal overlay">
+//           <div className="modal-background has-background-white-ter" />
+//           <div className="loader" />
+//         </div>
+//       </div >
```
Comment (anastasiiavorobiova) [inline src/components/TodoItem/TodoItem.tsx:267]: Consider removing redundant code here

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! The input field focus doesn't work properly, try to fix it and feel free to ask for help in the chat

**Comment by DenisGurskiy** [comment]
In VSCode all test pased.

**Review by etojeDenys — APPROVED** [review]
awesome

**Pull request closed without merging**